backup/setup/schools.json

```diff
@@ -299,5 +299,25 @@
 	"customYears": [],
 	"rssFeeds": [],
 	"__v": 0
+},
+{
+	"_id": {
+		"$oid": "5fda01df490123cba891a193"
+	},
+	"documentBaseDirType": "",
+	"systems": [],
+	"experimental": false,
```

Comment:
Show keys i think we have on key for generic markers and mark it as tombstone.

Reply:
Added `purpose`